src/DataProtection/DataProtection/src/KeyManagement/XmlKeyManager.cs

```diff
@@ -463,27 +465,25 @@ IAuthenticatedEncryptorDescriptor IInternalXmlKeyManager.DeserializeDescriptorFr
         }
     }
 
-    [UnconditionalSuppressMessage("Trimmer", "IL2057", Justification = "Type.GetType result is only useful with types that are referenced by DataProtection assembly.")]
     private IAuthenticatedEncryptorDescriptorDeserializer CreateDeserializer(string descriptorDeserializerTypeName)
     {
         var resolvedTypeName = TypeForwardingActivator.TryForwardTypeName(descriptorDeserializerTypeName, out var forwardedTypeName)
```

Comment:
And I think we pass the un-forwarded name to the activator in case it had special handling for the original name that would get skipped if we just instantiated the type we've already resolved?